### DISCUSSION TITLE
Who do you want to share with and why?

### DISCUSSION THREAD

**Discussion by MrCoder** [ZenUml/web-sequence#496] [Polls]
Sorry for the clickbait 😸 . We have not implemented this yet because we are not sure how you want us to do.

I 🤖 have a few questions:
1. Do you want to share your work or just want to check out what this button does?
2. If you want to share, do you want to share it publicly 📢 (e.g. on blog or social media) or privately 🔐 (e.g. co-worker)?
3. Did you share your work before? How did you do it? Screenshot or Download Image? Or something else?

**Comment by eqvis** [comment]
Hi,

I would like to share the diagram and DSL with a colleague, so that he can look at it and possibly modify anything...

Thank you for your work.
Best regards,
Mayra

**Reply by shab15** [reply]
Great overview! If you're looking to generate diagrams directly from structured text like YAML or JSON, Code2Diagram [https://code2diagram.com/console](javascript:void(0);) is a handy tool. It supports various formats including UML, Gantt charts, and more.



Reply

**Comment by dhochee-bardavon** [comment]
Similarly, I would like to share my diagram with others on my team. Ideally, they could edit, but it would be fine if they could just view the current state. 

For the view-only scenario, it would be great if the link I gave them once enabled them to view the current state later on. This could be super helpful for embedding into Jira stories.

**Comment by strandbygaard** [comment]
Using it on a dev team. Need to be able to collaborate on diagrams in team. Sharing DSL through git is too many additional steps. Needs to be click-edit-save.

**Comment by MrCoder** [comment]
Thank you to everyone who has provided feedback. We are now moving forward with plans to implement this feature. To ensure that we are prioritising your needs correctly, would you kindly inform us of the most critical targets that you aim to achieve with this.

- 🎉 Confluence  
- ❤️ Jira  
- 🚀  Slack 
- 👀  Other (please comment)

**Reply by Isammoc** [reply]
GitHub/GitLab

To be able to document as code and work with PR to change diagram. (Useful during an evolution)

To replace or in concurrence to mermaid rendering

**Reply by MrCoder** [reply]
I'm unsure about the technical implementation of this. A Chrome extension or a GitLab plugin?
